tendermint/src/trust_threshold.rs

```diff
@@ -42,15 +50,47 @@ impl TrustThresholdFraction {
     /// numerator are valid.
     ///
     /// The parameters are valid iff `1/3 <= numerator/denominator <= 1`.
-    /// In any other case we return `None`.
-    pub fn new(numerator: u64, denominator: u64) -> Option<Self> {
-        if numerator <= denominator && denominator > 0 && 3 * numerator >= denominator {
-            Some(Self {
-                numerator,
-                denominator,
-            })
-        } else {
-            None
+    /// In any other case we return an error.
+    pub fn new(numerator: u64, denominator: u64) -> Result<Self, Error> {
+        if numerator > denominator {
```

Comment:
```suggestion
        if numerator >= denominator {
```
I believe that the `=1` case is ambiguous here because a `1/3` threshold would require `+1/3` of the total voting power and correspondingly a threshold `=1` would require `>1` of the total voting power. What do you think?

Reply:
https://github.com/informalsystems/ibc-rs/issues/1149

Reply:
Done